contracts/packages/protocol/src/trade.rs

```diff
@@ -62,9 +62,7 @@ pub struct SwapMsg {
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
-pub struct Swap {
-    pub offer_asset: Asset,
-}
+pub struct Swap {}
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 pub struct Asset {
```

Comment:
It seems that you are no longer using this struct, maybe you can delete it if you don't intend to use it in the future :)

Reply:
Good catch, fixed at https://github.com/Local-Money/localmoney/pull/64/commits/6f0cdb9346d52348f02c93fb8ec00747d161b2fc